.github/workflows/ci.yml

```diff
@@ -150,10 +148,10 @@ jobs:
 
     - uses: actions/setup-python@v2
       with:
-        python_version: pypy3.7
+        python-version: pypy-3.7
```

Comment:
This typo was hidden by the fact nox does not error on missing interpreters (even when nothing runs). We were running only on ubuntu. Now at least we are running on macOS. Will have to figure out how to get something working on Windows eventually.